google/cloud/pubsub/doc/pubsub-main.dox

```diff
@@ -120,13 +120,13 @@ Follow these links to find examples for other `*Client` classes:
 
 /*! @page TopicAdminClient-endpoint-snippet Override @c TopicAdminClient Default Endpoint
 
-@snippet google/cloud/pubsub/samples/client_samples.cc topic-admin-client-set-endpoint
+@snippet google/cloud/pubsub/admin/samples/topic_admin_client_samples.cc topic-admin-client-set-endpoint
```

Comment:
o, these things are just called `set-client-endpoint`, not `foo-client-set-endpoint`

https://github.com/googleapis/google-cloud-cpp/blob/3f8954c9c3f1394e969b2412be0a4ad776805314/google/cloud/pubsub/admin/samples/topic_admin_client_samples.cc#L40

Reply:
ack, done